### PR TITLE
[Feature Extraction] Group 3-5 Feature

### PR DESCRIPTION
**Pull request by Kathryn-cat**
Added group 3-5 feature for block-level feature extraction.

### CODE REVIEW

File: src/meta_schedule/feature_extractor/per_block_feature.cc [hunk 1]
```diff
@@ -254,16 +254,192 @@ Feature::ForKindFeature::ForKindFeature(const ForVec& loops) {
...
+
+  explicit Feature(const LoopNest& loop_nest, const BlockRealizeNode* realize,
+                   arith::Analyzer* analyzer) {
+    for (const Buffer& buffer : realize->block->alloc_buffers) {
+      std::vector<int64_t> shape = utils::GetBufferShape(buffer, analyzer);
+      int64_t numel = 1;
+      for (int64_t x : shape) {
+        numel *= x;
+      }
+      const runtime::StorageScope storage_scope = runtime::StorageScope::Create(buffer.scope());
```
Comment (junrushao) [inline src/meta_schedule/feature_extractor/per_block_feature.cc:379]: nit: no need for the const qualifier. the compiler could identify constants and do constant-related optimization automatically.

```suggestion
     runtime::StorageScope storage_scope = runtime::StorageScope::Create(buffer.scope());
```

**Review by junrushao — APPROVED** [review]
LGTM

**Comment by junrushao** [comment]
I'm merging this PR given there is no actual problem - you may address those nitpicks in the next PR

File: src/meta_schedule/feature_extractor/per_block_feature.cc (hunk 1 above)
Comment (Kathryn-cat) [reply src/meta_schedule/feature_extractor/per_block_feature.cc:379]: Addressed :)

**Pull request merged**